### PR TITLE
Apb 792 find by arn

### PR DESCRIPTION
**Pull request by PaulHodgson**

### CODE REVIEW

File: app/uk/gov/hmrc/agentmapping/controller/MappingController.scala [hunk 1]
```diff
@@ -44,4 +45,10 @@ class MappingController @Inject()(mappingRepository: MappingRepository, desConne
       case _ => Future successful Forbidden
     }
   }
+
+  def findMapping(arn: uk.gov.hmrc.agentmtdidentifiers.model.Arn) = Action.async { implicit request =>
```
Comment (frankoid) [inline app/uk/gov/hmrc/agentmapping/controller/MappingController.scala]: should this be called findMappings (plural)?

File: it/uk/gov/hmrc/agentmapping/MappingControllerISpec.scala [hunk 2]
```diff
@@ -22,23 +23,29 @@ class MappingControllerISpec extends UnitSpec with MongoApp with WireMockSupport
...
   private val saAgentReference = "A1111A"
-  val request = createRequest()
+  val createMappingRequest: Resource = createMappingRequest()
 
-  def createRequest(requestUtr: Utr = utr, requestArn: Arn = registeredArn): Resource = {
+  def createMappingRequest(requestUtr: Utr = utr, requestArn: Arn = registeredArn): Resource = {
     new Resource(s"/agent-mapping/mappings/${requestUtr.value}/${requestArn.value}/$saAgentReference", port)
   }
 
+  def findMappingRequest(requestArn: Arn = registeredArn): Resource = {
```
Comment (frankoid) [inline it/uk/gov/hmrc/agentmapping/MappingControllerISpec.scala]: findMapping*s*Request?

File: app/uk/gov/hmrc/agentmapping/controller/MappingController.scala (hunk 1 above)
Comment (PaulHodgson) [inline app/uk/gov/hmrc/agentmapping/controller/MappingController.scala]: yes

File: it/uk/gov/hmrc/agentmapping/MappingControllerISpec.scala (hunk 2 above)
Comment (PaulHodgson) [inline it/uk/gov/hmrc/agentmapping/MappingControllerISpec.scala]: and yes

**Pull request merged**